### DISCUSSION TITLE
Paraview Phases

### DISCUSSION THREAD

**Discussion by javierpagalo** [PMEAL/OpenPNM#2180] [Q&A]
Hello
I'm an undergraduate student
I would like to know if there is a way to see the phases in paraview.
Thank you

**Comment by jgostick** [comment]
Hi @javierpagalo.  Your question is a bit vague, but I will try to answer based on my best guess of your problem.  When you 'export' to vtk, you have include a list of which phases you want to include in the exported filed.  By default, we do not export any phases, since this might lead to a messy file if you have a lot of phases defined.
